app.py

```diff
@@ -875,10 +878,10 @@ def tree_process(tree : ParentedTree, senttok: List[str]) -> tuple[ParentedTree,
 		# condition 2: label is punctuation sequence (which may or may not be the correct one for the token) + "-p"
 		# condition 3: token is unabiguously punctuation
 		# -> assign the appropriate punctuation label (either from PUNCT_TAGS or the default SYMBOL_TAG)
-		if is_possible_punct_sequence(subt.label) or is_punct_label(subt.label) or (is_possible_punct_sequence(senttok[i]) and senttok[i] not in AMBIG_SYM):
+		if is_punct_postag(subt.label) or is_punct_label(subt.label) or (is_possible_punct_token(senttok[i]) and senttok[i] not in AMBIG_SYM):
```

Comment:
please update the comments above

Reply:
https://github.com/nschneid/activedop/pull/59/commits/c4a65b151db6567df5366eea12f8488546cf5fa4